org.civicrm.styleguide/CRM/Styleguide/Manager.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+class CRM_Styleguide_Manager {
+
+  /**
+   * @var array
+   */
+  private $all = NULL;
+
+  /**
+   * Get the definition of specific style-guide by name.
+   *
+   * @param string $name
+   * @return array|NULL
+   *   If the style-guide exists, it is an array with properties:
+   *     - name: string
+   *     - label: string
+   */
+  public function get($name) {
+    $all = $this->getAll();
+    return @$all[$name];
```

Comment:
In its defense, I don't think this line is why `@` gets a bad rep (e.g. `@include "file"` would be a real problem), but thanks for calling it out since it's better to have a culture where `@` just doesn't exist. ;) I'll update it.